### PR TITLE
fix not display cmdwin when `cmdheight = 0`

### PR DESCRIPTION
**Pull request by hashue**
SSIA

### CODE REVIEW

File: autoload/ctrlp.vim [hunk 1]
```diff
@@ -330,6 +330,7 @@ fu! s:Open()
 endf
 
 fu! s:Close()
+  if &cmdheight != 0 | set cmdheight=0 |en
```
Comment (mattn) [inline autoload/ctrlp.vim:333]: Please use hard-tab.

File: autoload/ctrlp.vim [hunk 2]
```diff
@@ -2845,6 +2846,8 @@ fu! ctrlp#init(type, ...)
 	if shouldExitSingle && s:ExitIfSingleCandidate()
 		retu 0
 	en
+
+  if &cmdheight == 0 | set cmdheight=1 | en
```
Comment (mattn) [inline autoload/ctrlp.vim:2850]: ditto

**Comment by hashue** [comment]
Thaks for review. I fixed it.

**Comment by mattn** [comment]
Thank you

**Pull request merged**